pomodoro/src/firebase-config.js

```diff
@@ -0,0 +1,11 @@
+import * as firebase from 'firebase';
+
+export const firebaseConfig = {
+    apiKey: "AIzaSyCkaAsacq8_zHAq6Nz4yVAJYgEmfumzQ10",
```

Comment:
BOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO we don't commit API keys